metadata/src/main/java/org/apache/kafka/controller/ReplicationControlManager.java

```diff
@@ -947,6 +948,7 @@ ControllerResult<Map<Uuid, ApiError>> deleteTopics(ControllerRequestContext cont
                 results.put(id, ApiError.fromThrowable(e));
             }
         }
+        log.info("DeleteTopics result(s): {}", results);
```

Comment:
Can we do something like we do with create topics, where we build up a string like:

```DeleteTopics result(s): {id: p7foLAwZT8yDWn_yzHcYlg, name: foo, result:SUCCESS}, {id:EBxjGbySRfO1tmo13ZVUfA, name: bar, result: UNKNOWN_TOPIC_ID}```